### PR TITLE
Improve front matter example in page structures templates 

### PR DESCRIPTION
**Pull request by manjuu-eater**
### Description

Comment outed "..." to "# ..." which is not part of yaml

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/compatibility_tables/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For example, {{domxref("AbortController")}} would be added as shown below:
 ---
 title: AbortController
 slug: Web/API/AbortController
-…
+# …
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/mdn/writing_guidelines/page_structures/compatibility_tables/index.md:34]: If we do this, then we should change the "multiple compatibility" section below also:

https://github.com/mdn/content/blob/fb4083792de6982973a314f5d6fa9e4df2562580/files/en-us/web/api/channel_messaging_api/index.md?plain=1#L1-L8

[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/mdn/writing_guidelines/page_structures/compatibility_tables/index.md:34]: As we remove the field `tags:`, I think we should do this. That way, the YAML head matches what we actually do without editing it: easiest cut and paste.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by manjuu-eater** [comment]
Thank you for reviewing!

**Comment by bsmth** [comment]
Thanks @manjuu-eater and @teoli2003 👍🏻

**Pull request merged**